.travis.yml

```diff
@@ -24,8 +24,16 @@ matrix: # Add a separate config to the matrix, using clang as compiler
     - env: ROS_DISTRO=melodic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=moveit.rosinstall
       compiler: clang
 
+# Install lxml required for create_ikfast_moveit_plugin.py
+install:
+  - sudo apt-get install -y python-lxml
+
 before_script:
   - git clone -q --depth=1 https://github.com/ros-planning/moveit_ci.git .moveit_ci
+# Configure ikfast plugins for Fanuc and Panda
+  - git clone -q --depth=1 https://github.com/ros-planning/moveit_resources /tmp/resources
+  - moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.sh --quiet --name fanuc --pkg $PWD/fanuc_ikfast_plugin /tmp/resources/fanuc_description/urdf/fanuc.urdf manipulator base_link tool0
+  - moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.sh --quiet --name panda --pkg $PWD/panda_ikfast_plugin /tmp/resources/panda_description/urdf/panda.urdf panda_arm panda_link0 panda_link8
```

Comment:
i would like to see all of this noise (line 29, 33-36) moved inside moveit_ci in a script called ``test_ikfast.sh`` similar to how we have ``check_clang_format.sh``

Reply:
That's what I already suggested in https://github.com/ros-planning/moveit/pull/1320#issuecomment-456984311 and I'm awaiting your approval for https://github.com/ros-planning/moveit_ci/pull/42 ;-)

Reply:
Ok so this will go away after the moveit_ci change? great!